sql/src/main/java/io/crate/action/sql/Session.java

```diff
@@ -314,6 +316,8 @@ public DescribeResult describe(char type, String portalOrStatement) {
                 AnalyzedStatement analyzedStmt = portal.boundOrUnboundStatement();
                 if (analyzedStmt instanceof AnalyzedRelation) {
                     return new DescribeResult(((AnalyzedRelation) analyzedStmt).fields());
+                } else if (analyzedStmt instanceof AnalyzedUpdateStatement) {
```

Comment:
`AnalyzedRelation` and `AnalyzedUpdateStatement` and also later `AnalyzedInsertStatement` could share an interface marking these 3 classes that they have outputfields to simplify the code here.

Reply:
Sounds reasonable, I think `ResultSetStatement`  describes the iface better (maybe we can come up with an even better name?).